libs/blocks/merch-card/merch-card.js

```diff
@@ -306,6 +328,13 @@ const decorateFooterRows = (merchCard, footerRows) => {
   }
 };
 
+const setMiniCompareOfferSlot = (merchCard, offers) => {
+  if (merchCard.variant !== MINI_COMPARE_CHART) return;
+  const miniCompareOffers = createTag('div', { slot: 'offers' }, offers);
+  if (offers === undefined) { miniCompareOffers.appendChild(createTag('p')); }
```

Comment:
The two lines above can be combined into one I think:
```
const miniCompareOffers = createTag('div', { slot: 'offers' }, offers ?? createTag('p'));
```

Reply:
You are correct, it's done.